### ISSUE TITLE
File suffix has no dot

### ISSUE THREAD

**Issue by beatngu13** [retest/recheck#42]
As of `v0.4.3` (which is essentially [this commit](https://github.com/retest/recheck/commit/ae163b0278a71128923b391a5d44850948ea0ffe)), replay results are not properly persisted as they miss the suffix dot (e.g. `replayresult` instead of `replay.result`).

Maybe caused by this `TODO` from `RecheckImpl` (closed source):

```java
public File getResultFile() {
	return fileNamerStrategy.createFileNamer( execSuiteName )
	// TODO Remove after next release of recheck
	.getResultFile( ReportFileUtils.REPORT_FILE_EXTENSION.substring( 1 ) );
}
```

**Comment by roesslerj** [comment]
This is the wrong order ... closed source retest depends on recheck. So as soon as the parent-pom is switched, and we it depends on recheck 0.4.3 (which it doesn't as of now), this bug is activated. Since Enes did this all locally, he triggered that bug. Good catch, though.

**Comment by roesslerj** [comment]
Dammit, I was too tired. Yes, this is a problem that needs to be addressed... reopened until fixed.

**Comment by beatngu13** [comment]
How about a Hotfix of retest with this `TODO` being addressed, that should do the job?

**Comment by beatngu13** [comment]
Fixed with the release of retest 3.0.1.
